src/Pages/home/HomePage.tsx

```diff
@@ -1,131 +1,95 @@
-                  className="carImg"
-                />
-                <Image src={blueBmw} alt="Avatar" className="avatar"></Image>
-                <Card.Body className="cardText">
-                  <Card.Title>David beckam</Card.Title>
-                  <Card.Text>5 year of experience</Card.Text>
-                  <Card.Text>Hyundai Creata</Card.Text>
-
-                  <Card.Text>25 / Hour</Card.Text>
-                </Card.Body>
-                <Button variant="outline-primary">Book</Button>
-              </Card>
-            </Col>
-            <Col sm className="colMargin">
-              <Card className="cardHeight">
-                <Card.Img
-                  variant="top"
-                  src={roberthaverly}
-                  className="carImg"
-                />
-                <Image src={blueBmw} alt="Avatar" className="avatar"></Image>
-                <Card.Body className="cardText">
-                  <Card.Title>David beckam</Card.Title>
-                  <Card.Text>5 year of experience</Card.Text>
-                  <Card.Text>Hyundai Creata</Card.Text>
-
-                  <Card.Text>25 / Hour</Card.Text>
-                </Card.Body>
-                <Button variant="outline-primary">Book</Button>
-              </Card>
-            </Col>
-            <Col sm className="colMargin">
-              <Card className="cardHeight">
-                <Card.Img
-                  variant="top"
-                  src={roberthaverly}
-                  className="carImg"
-                />
-                <Image src={blueBmw} alt="Avatar" className="avatar"></Image>
-                <Card.Body className="cardText">
-                  <Card.Title>David beckam</Card.Title>
-                  <Card.Text>5 year of experience</Card.Text>
-                  <Card.Text>Hyundai Creata</Card.Text>
-
-                  <Card.Text>25 / Hour</Card.Text>
-                </Card.Body>
-                <Button variant="outline-primary">Book</Button>
-              </Card>
-            </Col>
-            <Col sm className="colMargin">
-              <Card className="cardHeight">
-                <Card.Img
-                  variant="top"
-                  src={roberthaverly}
-                  className="carImg"
-                />
-                <Image src={blueBmw} alt="Avatar" className="avatar"></Image>
-                <Card.Body className="cardText">
-                  <Card.Title>David beckam</Card.Title>
-                  <Card.Text>5 year of experience</Card.Text>
-                  <Card.Text>Hyundai Creata</Card.Text>
-
-                  <Card.Text>25 / Hour</Card.Text>
-                </Card.Body>
-                <Button variant="outline-primary">Book</Button>
-              </Card>
-            </Col>
-          </Row>
-        </CardDeck>
-        <Link to="./login">login</Link>
-      </div>
-    );
+    console.log("this is state", this.state);
+    if (this.state.loading === true) {
```

Comment:
the if statement takes true or false and the value of loading is either true of false.

so you dont have to equal it to true to check if it is true